proto/control-api/src/grpc/mod.rs

```diff
@@ -5,7 +5,5 @@
     clippy::pedantic,
 )]
 
-pub mod api;
-pub mod api_grpc;
-pub mod callback;
-pub mod callback_grpc;
+pub mod medea;
+pub mod medea_callback;
```

Comment:
These should be named _just_ `api` and `callback`.

Reply:
Repeating `medea`/`jason` everywhere has a little sense. It's more than enough to have them in crate's name. No need for repetition.